web/src/LogStore.ts

```diff
@@ -293,12 +368,20 @@ class LogStore {
     return this.logHelper(spans, 0).lines
   }
 
+  allSpans(): { [key: string]: LogSpan } {
+    const result: { [key: string]: LogSpan } = {}
+    for (let spanId in this.spans) {
+      result[spanId] = { ...this.spans[spanId] }
```

Comment:
Removed the clone - it wasn't strictly necessary, I just did it to avoid callers unintentionally mutating internal `LogStore` state. Not as much of a concern now that there's way less logic with the result of this now.